jobsapp/models.py

```diff
@@ -1,42 +1,94 @@
+# DJANGO Imports
 from django.db import models
 from django.utils import timezone
+from django.utils.translation import ugettext as _
 
+# APP Imports
```

Comment:
I wouldn't add these comments in the imports because we are using isort to sort them and I don't know if it will do the correct job if there are comments in between.

Reply:
Fix this @xdiume and we'll merge it :)

Reply:
I fixed it while resolving merge conflicts, merging!